business_requirement_deliverable_cost/models/business.py

```diff
@@ -98,12 +100,12 @@ def product_uom_change(self):
                 self.product_id.uom_id.id
             ) / self.qty
 
-        if pricelist:
+        if pricelist_id:
```

Comment:
@elicoidal is like you said.

onchange product for DL is taken `product.list_price`
onchange product for RL is taken `product.list_price`